### PR TITLE
Add reposts + purchased albums to library PAY-1679

### PR DESCRIPTION
**Pull request by nicoback2**
### Description
Use new audius query hook to add all library albums including reposts and purchased to library
Note this is being merged into feature branch, see accompanying main PR here: 
https://github.com/AudiusProject/audius-client/pull/3864

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/web/src/pages/saved-page/components/desktop/SavedPage.module.css [hunk 1]
```diff
@@ -115,3 +115,13 @@
...
 }
+
+.spinner {
+  height: 40px;
+  width: 40px;
+  margin: 0 auto;
+}
+
+.spinner g path {
+  stroke: var(--secondary) !important;
```
Comment (schottra) [inline packages/web/src/pages/saved-page/components/desktop/SavedPage.module.css:126]: Is the `!important` actually needed? I just styled a loading spinner in another PR and it took the stroke color without it.
Also we should really encapsulate the color/sizes for LoadingSpinner into the component the same way we do for `Icon`

**Review by schottra — APPROVED** [review]

File: packages/common/src/store/cache/collections/utils/index.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// TODO: Migrate utils from packages/web/common/store/cache/collections/utils to here
```
Comment (schottra) [inline packages/common/src/store/cache/collections/utils/index.ts:1]: Is this a leftover comment or something for the future? If it's for the future, do you mind making a linear item and adding the link here?

**Review by schottra — APPROVED** [review]
Looks good!

File: packages/common/src/store/cache/collections/utils/index.ts (hunk 2 above)
Comment (nicoback2) [reply packages/common/src/store/cache/collections/utils/index.ts:1]: Yess. Basically the task is to finish the common migration, will update this comment. https://linear.app/audius/issue/C-1280/migrate-remaining-webcommon-to-common

**Pull request merged**